nixos/modules/services/web-servers/tomcat.nix

```
@@ -127,7 +127,7 @@ in
       webapps = mkOption {
         type = types.listOf types.path;
         default = [ tomcat.webapps ];
-        defaultText = literalExpression "[ pkgs.tomcat85.webapps ]";
+        defaultText = literalExpression "[ pkgs.tomcat9.webapps ]";
```

```suggestion
        defaultText = literalExpression "[ config.services.tomcat.package.webapps ]";
```
is probably better here than the text that was there originally. the old text gave the impression that changing `package` did not also change where this option pulls its value.

---

That's indeed not a bad suggestion. I've just amended the commit again :)